util/include/DigestImpl.ipp

```diff
@@ -133,8 +139,6 @@ class OpenSSLDigestCreator : public DigestCreator {
   void final() override {}
 
  private:
-  SHACTX hash_ctx_;
+  EVP_MD_CTX* hash_ctx_;
 };
-}  // namespace concord::util::digest
-
-//#include "DigestImpl.ipp"
\ No newline at end of file
+}  // namespace concord::util::digest
```

Comment:
Better to add a new line.

Reply:
Addressed https://github.com/arc-vmware/concord-bft/pull/2/commits/be7b4d3f850359b235887ae82cbe8f4c29a080c4